### DISCUSSION TITLE
Public link is null

### DISCUSSION THREAD

**Discussion by Sammers21** [tdlight-team/tdlight-java#193] [Q&A] [answered]
After doing `GetSupergroupFullInfo` on `supergroup id` = 1315078272(which is https://t.me/temalebedev), i'm getting null in the invtite link field:

![image](https://github.com/tdlight-team/tdlight-java/assets/16746106/ac2151fe-490a-4244-82c6-5b8486a780c2)

Is that expected behaviour? Is there a way to get the invite link? The search results(with `SearchPublicChat`) does with contain channel username or a link either.

**Comment by cavallium** [comment]
Always read the documentation! That field is only shown when you are the administrator of that group

**Comment by Sammers21** [comment]
@cavallium, sure. But i still have no idea how i can get the channel username. Is it even possible with tdlib?

**Reply by cavallium** [reply, marked as answer]
Call getSupergroup
